src/server/pfs/server/api_server.go

```diff
@@ -599,6 +595,27 @@ func (a *apiServer) DeleteAll(ctx context.Context, request *types.Empty) (respon
 	return &types.Empty{}, nil
 }
 
+// V2 Methods
+// GetTar not implemented
+func (a *apiServer) GetTar(request *pfs.GetTarRequest, server pfs.API_GetTarServer) (retErr error) {
+	return errors.Errorf("method not implemented")
```

Comment:
Probably could use a more descriptive message that mentions that this is a V2 rpc.